### PR TITLE
Cockpit 271

### PR DESCRIPTION
**Pull request by skobyda**
Preview: https://skobyda.github.io/cockpit-project.github.io/blog/cockpit-271.html

### CODE REVIEW

**Comment by skobyda** [comment]
This PR is a clone of #533, after I accidentally closed it :)

File: _posts/2022-06-08-cockpit-271.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+tags: cockpit
+slug: cockpit-271
+category: release
+summary: 'Container names on the metrics page'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit-podman 49.1:
```
Comment (martinpitt) [inline _posts/2022-06-08-cockpit-271.md:14]: Please mention all projects.

File: _posts/2022-06-08-cockpit-271.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+## Show container names on the Metrics page
```
Comment (martinpitt) [inline _posts/2022-06-08-cockpit-271.md:17]: "Metrics: Show container names" for our usual format.

File: _posts/2022-06-08-cockpit-271.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/271)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-4869d82c66)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/FEDORA-2022-80d2aa8e21)
+* [Cockpit Client](https://flathub.org/apps/details/org.cockpit_project.CockpitClient)
+* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/270.1)
+* [cockpit-machines Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-d30c57494e)
+* [cockpit-machines Fedora 35](https://bodhi.fedoraproject.org/updates/FEDORA-2022-f584409196)
+* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/49.1)
+* [cockpit-podman Fedora 36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-dcc6dd5d17)
+* [cockpit-podman Fedora 35](https://bodhi.fedoraproject.org/updates/FEDORA-2022-f91d6f8df9)
```
Comment (martinpitt) [inline _posts/2022-06-08-cockpit-271.md:42]: These are the obsolete updates for the 49/270 versions, replaced with .1 versions.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks!

File: _posts/2022-06-08-cockpit-271.md (hunk 1 above)
Comment (martinpitt) [reply _posts/2022-06-08-cockpit-271.md]: Ah, good point -- well, matter of taste I suppose.

**Review by garrett — APPROVED** [review]
Hey! I can approve this, as this PR isn't one I opened! :wink: 

I rewrote the text a while back on the original PR and it shows up here. It still looks fine to me. I'm happy if anyone can find some issues we can fix, of course. But I'm fine with it as-is. Screenshot looks fine too.

It passes the checklist as well: Links have been updated and summary has been added.

:+1:

**Pull request merged**